rust/libnewsboat/src/utils.rs

```diff
@@ -43,8 +43,8 @@ pub fn consolidate_whitespace(input: String) -> String {
     result
 }
 
-pub fn to_u(rs_str: String, default_value: u32) -> u32 {
-    let mut result = rs_str.parse::<u32>();
+pub fn to_u(s: String, default_value: u32) -> u32 {
```

Comment:
I don't think it's useful to change one nondescript name to another nondescript one. If the argument *must* be renamed, I'd rather pick a useful name (e.g. "number_str" in this case).

Reply:
For me, since it's rust. We should not use `number_str`, that is a place I feel weird to have redundant variable names when the language supports shadowing. Maybe I can do that in another commit later on when I got the idea on how it should be renamed, I do it so that others do not just copy and paste names that is even worse around.

Reply:
Sorry, this makes no sense to me. Nothing is getting shadowed here as far as I can see. Can you explain some other way?

I agree the copy-paste is a bit of a problem, but with short functions, it's pretty harmless, so I tolerate it there.

Reply:
No, I don't mean in this case. But generally,
```rust
let data = something;
let data = data.ops;
```
is better than
```rust
let data_str = something;
let data_result = data.ops;
```

Reply:
Okay, got that. But what is the rationale for *this* case? I still think this whole commit should be reverted.

Reply:
At least it is less redundant, but the naming could be changed later. Feel free to revert it.